contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/HttpBatchReader.java

```diff
@@ -66,7 +67,7 @@ public class HttpBatchReader implements ManagedReader<SchemaNegotiator> {
   private final int maxRecords;
   protected final Paginator paginator;
   protected String baseUrl;
-  private JsonLoader jsonLoader;
+  private JsonLoaderImpl jsonLoader;
```

Comment:
I hadn't noticed that we'd already started depending on JsonLoaderImpl. Is something preventing working with the JsonLoader interface?